arm-web/2015-08-01/swagger/AppServiceCertificateOrders.json

```diff
@@ -821,47 +821,36 @@
     "AppServiceCertificate": {
```

Comment:
Thanks Navy. It seems we are duplicating properties in these two models, i just had a discussion with @amarzavery and confirmed that we could share properties here as:


```json
   "definitions": {
      "AppServiceCertificateResource": {
        "description": "Key Vault container ARM resource for a certificate that is purchased through Azure.",
        "type": "object",
        "allOf": [
          {
            "$ref": "#/definitions/Resource"
          }
        ],
        "properties": {
          "properties": {
            "$ref": "#/definitions/AppServiceCertificate",
            "x-ms-client-flatten": true
          }
        }
      },
      "AppServiceCertificate": {
        "description": "ARM envelope properties",
        "properties": {
          "keyVaultId": {
            "description": "Key Vault Csm resource Id.",
            "type": "string"
          },
          "keyVaultSecretName": {
            "description": "Key Vault secret name.",
            "type": "string"
          },
          "provisioningState": {
            "description": "Status of the Key Vault secret.",
            "enum": [
              "Initialized",
              "WaitingOnCertificateOrder",
              "Succeeded",
              "CertificateOrderFailed",
              "OperationNotPermittedOnKeyVault",
              "AzureServiceUnauthorizedToAccessKeyVault",
              "KeyVaultDoesNotExist",
              "KeyVaultSecretDoesNotExist",
              "UnknownError",
              "ExternalPrivateKey",
              "Unknown"
            ],
            "type": "string",
            "readOnly": true,
            "x-ms-enum": {
              "name": "KeyVaultSecretStatus",
              "modelAsString": false
            }
          }
        }
      }
    }
```